pkg/webhook/webhook.go

```diff
@@ -75,9 +76,37 @@ func validateIngressNodeFirewall(ctx context.Context, inf *ingressnodefwv1alpha1
 			schema.GroupKind{Group: ingressnodefwv1alpha1.GroupVersion.Group, Kind: ingressnodefwv1alpha1.IngressNodeFirewall{}.Kind},
 			inf.Name, allErrs)
 	}
+	if allErrs := validateINFInterfaces(ctx, inf.Spec.Interfaces, inf.Name, kubeClient); len(allErrs) > 0 {
+		return apierrors.NewInvalid(
+			schema.GroupKind{Group: ingressnodefwv1alpha1.GroupVersion.Group, Kind: ingressnodefwv1alpha1.IngressNodeFirewall{}.Kind},
+			inf.Name, allErrs)
+	}
 	return nil
 }
 
+func validateINFInterfaces(ctx context.Context, infInterfaces []string, infName string, kubeClient client.Client) field.ErrorList {
+	var allErrs field.ErrorList
+
+	for idx, inf := range infInterfaces {
+		if inf == "" {
+			allErrs = append(allErrs,
+				field.Invalid(field.NewPath("spec").Child("interfaces").Index(idx).Key("interaces"),
+					infName, fmt.Sprintf("must be valid interface names %s", inf)))
```

Comment:
Maybe let the user know blank interfaces arent ok - printing out the black inf is pointless.